portfolio/src/main/webapp/StyleMenu.css

```diff
@@ -0,0 +1,94 @@
+body{
+    background: whitesmoke;
+
+}
+#socials {
+  margin:0;
+  color: #dbd8d8;
+}
+.projects {
+  background: #a8a7a75e;
+  background-size: cover;
+  height: 600px;
+  position: relative;
+}
+#projects{
+    border: none;
+}
+.experience {
+    background: rgba(117, 117, 116, 0.623);
+  background-size: cover;
+  height: 600px;
+  position: relative;
+}
+#experience{
+    border: none;
+}
+.about {
+  background: rgba(8, 8, 8, 0.507); 
+  background-size: cover;
+  height: 500px;
+  position: relative;
+}
+.blog{
+  background: black; 
+  background-size: cover;
+  height: 500px;
+  position: relative;
+}
+.profile{
+   
+  display: inline-block;
+  width: 250px;
+  height: 250px;
+  border-radius: 50%;
+
+  background-repeat: no-repeat;
+  background-position: left center;
+  background-size: cover;
+}
+.picture1{
+   
+  display: inline-block;
+  width: 300px;
+  height: 300px;
+  border-radius: 50%;
+
+  background-repeat: no-repeat;
+  background-position: left center;
+  background-size: cover;
+}
+.picture2{
+   
+  display: inline-block;
+  width: 300px;
+  height: 300px;
+  border-radius: 50%;
+
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+.picture3{
+   
+  display: inline-block;
+  width: 300px;
+  height: 300px;
+  border-radius: 50%;
+
+  background-repeat: no-repeat;
+  background-position: right center;
+  background-size: cover; 
+}
+
+textarea{
+  margin-right: 600px;
+  text-align: left;    
+  width: 600px;
+  height: 300px;
+  background-color: black;
+  color: white;
+  position: absolute;
+  right: 0px;
+  
+}
```

Comment:
Add a new line to the end of this file so that github will be happy about the styling. It doesn't actually make a difference but that is just how the styling is for github.